amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -211,4 +220,18 @@ vx_status Nop_Register(vx_context);
 #define VX_KERNEL_RPP_CROP_MIRROR_NORMALIZE_NAME                "org.rpp.CropMirrorNormalize"
 #define VX_KERNEL_RPP_NOP_NAME                                  "org.rpp.Nop"
 
+// Tensor kernel names
+#define VX_KERNEL_RPP_NOP_NAME                                               "org.rpp.Nop"
+#define VX_KERNEL_RPP_COPY_NAME                                              "org.rpp.Copy"
+#define VX_KERNEL_RPP_BRIGHTNESS_NAME                                        "org.rpp.Brightness"
+#define VX_KERNEL_RPP_GAMMACORRECTION_NAME                                   "org.rpp.GammaCorrection"
+#define VX_KERNEL_RPP_CROPMIRRORNORMALIZE_NAME                               "org.rpp.CropMirrorNormalize"
+#define VX_KERNEL_RPP_RESIZE_NAME                                            "org.rpp.Resize"
+#define VX_KERNEL_RPP_CROP_NAME                                              "org.rpp.Crop"
+#define VX_KERNEL_RPP_COLORTWIST_NAME                                        "org.rpp.ColorTwist"
+#define VX_KERNEL_RPP_RESIZEMIRROR_NAME                                      "org.rpp.ResizeMirrorNormalize"
+#define VX_KERNEL_RPP_SEQUENCEREARRANGE_NAME                                 "org.rpp.SequenceRearrange"
```

Comment:
Line numbers 224 - 233 is not required in this PR, Please make sure to include only the changes for Audio

Reply:
removed